cypress/integration/FileInputFieldWithList/Disables_button_when_full.feature

```diff
@@ -0,0 +1,5 @@
+Feature: The FileInputFieldWithList disables its button when full
+
+    Scenario: The button is disabled when the fileInput is full
+        Given a FileInputFieldWithList without multiple and a single file is rendered
```

Comment:
This could/should be split into two steps:

```feature
Given a FileInputFieldWithList without multiple is rendered
And the FileInputFieldWithList holds a file
```